File: app/src/main/java/com/gustavoforero/moviesomdb/net/responses/SearchResponse.java
package com.gustavoforero.moviesomdb.net.responses;

import com.gustavoforero.moviesomdb.domain.Movie;

import java.util.ArrayList;

public class SearchResponse {
    private boolean Response;
    private String Error;
    private ArrayList<Movie> Search;

    public boolean isSuccess() {
        return Response;
    }

    public String getError() {
        return Error;
    }

    public ArrayList<Movie> getSearch() {
        return Search;
    }
}

File: app/src/main/java/com/gustavoforero/moviesomdb/net/responses/SearchResponse.kt
package com.gustavoforero.moviesomdb.net.responses

import com.gustavoforero.moviesomdb.domain.Movie

import java.util.ArrayList

class SearchResponse {
     var Response: Boolean = false
     var Error: String? = null
     var Search: ArrayList<Movie>? = null
}
